packages/react/src/components/InAppMessaging/BannerMessage/BannerMessage.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Heading,
+  Icon,
+  Image,
+  Text,
+  View,
+  Button,
+  Flex,
+} from '@aws-amplify/ui-react';
+
+import '../../../../src/styles.css';
+
+export default function BannerMessage({
+  position = 'top',
+  ...props
+}): JSX.Element | null {
+  const {
+    body,
+    header,
+    image,
+    layout,
+    onClose,
+    primaryButton,
+    secondaryButton,
+  } = props;
+
+  const hasPrimaryButton = primaryButton.title && primaryButton.title != '';
+  const hasSecondaryButton =
+    secondaryButton.title && secondaryButton.title != '';
+
+  return (
+    <Flex className={`container ${layout}`}>
+      <Flex className="contentContainer">
+        <View className="imageContainer">
+          <Image src={image.src} className="image" alt="Message Image" />
+        </View>
+        <Flex className="textContainer">
+          {header?.content && <Heading>{header.content}</Heading>}
```

Comment:
Prefer ternary over `&&` operator (can be applied through out this file)
```suggestion
{header?.content ? <Heading>{header.content}</Heading> : null}
```